### PR TITLE
[2.2]skip load segment if already loaded

### PR DESCRIPTION
**Pull request by weiliu1031**
/kind improvement

if segment already loaded, skip submit task to scheduler, just return success.

### CODE REVIEW

File: internal/querynode/impl.go [hunk 1]
```diff
@@ -485,6 +485,21 @@ func (node *QueryNode) LoadSegments(ctx context.Context, in *querypb.LoadSegment
 		return node.TransferLoad(ctx, in)
 	}
 
+	alreadyLoaded := true
+	for _, info := range in.Infos {
+		has, _ := node.metaReplica.hasSegment(info.SegmentID, segmentTypeSealed)
+		if !has {
+			alreadyLoaded = false
+		}
+	}
```
Comment (congqixia) [inline internal/querynode/impl.go:494]: cannot skip all segment info here, 2.2 has task merge logic

Comment (weiliu1031) [reply internal/querynode/impl.go:494]: only all segment has been loaded , it can skip

**Comment by weiliu1031** [comment]
rerun ut

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**